### DISCUSSION TITLE
What's the new syntax for maxpdf?

### DISCUSSION THREAD

**Discussion by jhiscocks** [mtex-toolbox/mtex#1066] [Ask Anything] [answered]
The maxpdf function used to take an odf and a miller index and provide the maximum of the pole figure corresponding to the miller index.  This command seems to have be obsolete now, is there a replacement?

**Comment by ralfHielscher** [comment, marked as answer]
Hi Jessica,

I think we never had a function ```maxPDF```. But you can do it with
```matlab
pdf = calcPDF(odf, Miller(1,0,0,odf.CS));
[value,pos] = max(pdf)
```
Ralf.

**Reply by jhiscocks** [reply]
Thank you!
